app/src/db/dataConnection.js

```diff
@@ -1,8 +1,9 @@
 const Knex = require('knex');
-const knexfile = require('../../knexfile');
-const log = require('npmlog');
 const { Model } = require('objection');
 
+const knexfile = require('../../knexfile');
+const log = require('../components/log')(module.filename);
```

Comment:
just checking.. would we want to include the log module globally instead of requiring on each file where it is used?

Reply:
The module.filename is used to let the log populate the metadata component attribute for each of the logs by default. It's a clean way to ensure we have a standard level of annotations for all our log statements for traceability reasons.